pkg/resources/schema_acceptance_test.go

```diff
@@ -501,6 +502,28 @@ func TestAcc_Schema_ManagePublic(t *testing.T) {
 					resource.TestCheckResourceAttr("snowflake_schema.test", "pipe_execution_paused", "false"),
 				),
 			},
+			// make sure the schema wasn't dropped
```

Comment:
why we calidate it on the import step and not on the first step after bumping the version? Also, shouldn't we check 0.94.0 -> 0.94.1 transition?

Reply:
Changed - now tests are separated for v0.94.0 and v0.94.1.